index.html

```diff
@@ -170,45 +170,44 @@ <h4 class="connect">LET’S CONNECT</h4>
-                                    <img src="./image/javascript.png" alt="">
+                                    <img src="./image/javascript.png" alt="javascript image">
                                     <p>JavaSript</p>
                               </li>
                               <li class="javascript">
-                                    <img src="./image/html.png" alt="">
+                                    <img src="./image/html.png" alt="html image">
                                     <p>HTML</p>
                               </li>
                               <li class="javascript">
-                                    <img src="./image/css.png" alt="">
+                                    <img src="./image/css.png" alt="css image">
                                     <p>CSS</p>
                               </li>
                               <li class="framework">
                                     <h4>Frameworks</h4>
-                                    <img src="./image/chevronright.png" alt="">
+                                    <img src="./image/chevronright.png" alt="chevronright">
                               </li>
                               <li class="framework">
                                     <h4>Skills</h4>
-                                    <img src="./image/chevronright.png" alt="">
+                                    <img src="./image/chevronright.png" alt="chevronright">
                               </li>
                         </ul>
                   </section>
             </section>
-            <div class="footer-end">
-                  <div class="footer">
-                        <h4>Contact me</h4>
-                        <p>If you have an application you are interested in developing, a feature that you need built or
-                              a project that needs coding. I’d love to help with it </p>
-                        <div class="input">
-                              <input type="text">
-                              <input type="email">
-                              <textarea name="text" cols="30" rows="10"
-                                    placeholder="Write your message here"></textarea>
-                        </div>
-                        <button class="getbutton">Get It Touch</button>
-                        <img src="./image/Shape.png" alt="">
-                  </div>
+      </main>
 
-            </div>
-      </section>
+      <footer class="footer-end">
+            <form action="https://formspree.io/f/xoqrojyw" method="post" class="footer">
+                  <h4>Contact me</h4>
+                  <p>If you have an application you are interested in developing, a feature that you need built or
+                        a project that needs coding. I’d love to help with it </p>
+                  <div class="input">
+                        <input type="text" name="name" placeholder="Name" required>
```

Comment:
- [ ] you should add the max length in order to make the name input only accept up to 30 characters as required in the requirements